commit 05f31ce8c605a564b049242230c3e22d8ca30a30 (refs/changes/18/2518/4)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2017-09-11T14:05:36-04:00 (2 years, 1 month ago)
    
    OS-6334 OS-816 forgot about ipsecah
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/usr/src/uts/common/inet/ip/ipsecah.c b/usr/src/uts/common/inet/ip/ipsecah.c
index b533d75398..7d5eac71b3 100644
--- a/usr/src/uts/common/inet/ip/ipsecah.c
+++ b/usr/src/uts/common/inet/ip/ipsecah.c
@@ -173,8 +173,8 @@ ah_kstat_init(ipsecah_stack_t *ahstack, netstackid_t stackid)
 	ipsec_stack_t	*ipss = ahstack->ipsecah_netstack->netstack_ipsec;
 
 	ahstack->ah_ksp = kstat_create_netstack("ipsecah", 0, "ah_stat", "net",
-	    KSTAT_TYPE_NAMED, sizeof (ah_kstats_t) / sizeof (kstat_named_t),
-	    KSTAT_FLAG_PERSISTENT, stackid);
+	    KSTAT_TYPE_NAMED, sizeof (ah_kstats_t) / sizeof (kstat_named_t), 0,
+	    stackid);
 
 	if (ahstack->ah_ksp == NULL || ahstack->ah_ksp->ks_data == NULL)
 		return (B_FALSE);
